scripts/sbatch_eval/readme.md

```diff
@@ -0,0 +1,25 @@
+
+# Basic Info
+
+The sbatch-script `run_megatron_server.sbatch` is designed to perform evaluation of all current model checkpoints for tokenizer ablation, launching Megatron-LM inference server / LM-eval-harness client pairs.
+
+# Setup
+
+Make sure `run_server_no_opt.py` is present in the same directory as `run_megatron_server.sbatch`.\
+Obtain the current list of model checkpoint locations (`checkpoints.txt`) and place it in this directory.\
+Adjust the working directory and cache paths in `run_megatron_server.sbatch`.
+
+# Usage
+
+Run the script using
+
+```
+sbatch [sbatch_options] run_megatron_server.sbatch /path/to/checkpoints.txt "list,of,tasks" /path/to/output/directory batch_size
+```
+By default, the list of checkpoint paths to be evaluated is checkpoint_list.txt in the same directory as `run_checkpoints.sbatch`. \
```

Comment:
Please fix, the file name is `run_megatron_server_client.sbatch`.

Reply:
Also, there is no default path for `checkpoint_list.txt`. (It should be named `checkpoints.txt` according to the rest of the README.)